CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## in-toto v0.2.2
+
+* Add support for gpg signing subkeys.
+* Drop strict requirement on securesystemslib 0.9.
+* Command line tool changes:
+    - Add a --base-path parameter to in-toto record and in-toto run
+    - in-toto-record now follows symbolic links
+* Fixed typo in exception messages
+* Adds support for sublayout namespacing (for in-toto spec 0.9 compliance)
+* Path prefix is normalized during in-toto verification:
+    - Paths such as foo//bar match with foo/bar.
```

Comment:
Should you mention this change?
https://github.com/in-toto/in-toto/commit/7ad8942b6c9082f8e5773f8fb0f2ed3375b86f18

Reply:
Thanks for the heads up! will add